package.json

```diff
@@ -137,6 +137,8 @@
     "react-render-html": "^0.6.0",
     "react-router-dom": "^4.3.1",
     "react-spinners": "^0.5.1",
+    "react-star-rating-component": "^1.4.1",
+    "react-star-ratings": "^2.3.0",
```

Comment:
@myrdstom `react-start-ratings` doesn't seem to be used anywhere in your PR and yet you have added it as a dependency.
Could you kindly remove it or let us know why it is added. :-)